app/models/procedure_revision.rb

```diff
@@ -203,9 +203,9 @@ def coordinate_for(tdc)
   private
 
   def compute_estimated_fill_duration
-    tdc_durations = types_de_champ_public.fillable.map do |tdc|
-      duration = tdc.estimated_fill_duration(self)
-      tdc.mandatory ? duration : duration / 2
+    tdc_durations = types_de_champ_public.map do |tdc|
+      duration = tdc.estimated_fill_duration(self) + tdc.estimated_read_duration
+      tdc.mandatory || !tdc.fillable? ? duration : duration / 2
```

Comment:
je n'ai pas compris pourquoi un champ non remplissable avait sa durée de remplissage divisée par 2 sachant que son `estimated_fill_duration` est de 0

```suggestion
      tdc.mandatory ? duration : duration / 2
```

Reply:
Un champ "non fillable" ne peut pas être mandatory, ça n'a donc pas de sens de diviser /2 l'`estimated_read_duration` qu'on vient de calculer (l'`estimated_fill_duration` étant bien à 0 lui). Mais c'est pas clair du tout je m'étais emmêlé les pinceaux aussi, je refactorerai lundi.
J'en profiterai pour revoir un truc: inutile de tenir compte du temps de lecture des "libellés" des tdc: ils restent brefs par rapport au reste alors que ça rajoute vachement de complexité dans les tests.

Reply:
J'ai refacto, un poil plus long et redondant dans le code, mais un peu plus clair.